modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc

```diff
@@ -0,0 +1,102 @@
+[[azure-server-setup]]
+= Initial setup
+
+.Procedure: Login and update the system
+. SSH into your {azure} instance.
+
+. Switch to the root user and update the system.
++
+
+----
+sudo -i
+zypper refresh
+zypper update
+----
+
+== Network Configuration
+
+Before initiating `yast2 susemanager_setup`, ensure the network is correctly configured for {azure}.
+
```

Comment:
I would add here , that in Azure for the SUMA PAYG setup it is required, that a new Virtual Network and a new subnet is configured. (Azure requirement, not our).
So it is impossible to setup the instance in an existing network.

When you setup new clients which should be managed, take care that you put them into the private subnet which is configured with SUMA PAYG. (This is the section below).

If the customer want to manage systems in an already existing network, they can configure "peering".

Reply:
Thanks, I added that note.  I hope I got it right.